vim/syntax/tmux.vim

```diff
@@ -57,11 +57,13 @@ hi def link tmuxVariableExpansion Identifier
 
 " Make the foreground of colourXXX keywords match the color they represent.
 " Darker colors have their background set to white.
```

Comment:
Please update the comment with something like, "When tmux_syntax_colors is unspecified or set to a non-zero value, ..."

Reply:
Done. I also moved the changes from `vim/syntax/tmux.vim` to `src/header.vim`.